### ISSUE TITLE
R: depends on pango

### ISSUE THREAD

**Issue by Char-Aznable** [Homebrew/linuxbrew-core#12858]
**Please note we will close your issue without comment if you delete, do not read or do not fill out the issue checklist below and provide ALL the requested information. If you repeatedly fail to use the issue template, we will block you from ever submitting issues to Homebrew again.**

- [x] are reporting a bug others will be able to reproduce and not asking a question or requesting software. If you're not sure or want to ask a question do so on our Discourse: https://discourse.brew.sh. To get software added or changed in Homebrew please file a [Pull Request](https://github.com/Homebrew/linuxbrew-core/blob/master/CONTRIBUTING.md)
- [x] have a problem with `brew install` (or `upgrade`, `reinstall`) a single, Homebrew/homebrew-core formula (not cask) on macOS? If it's a general `brew` problem please file this issue at Homebrew/brew: https://github.com/Homebrew/brew/issues/new/choose. If it's a Linux problem please file this issue at https://github.com/Homebrew/linuxbrew-core/issues/new/choose. If it's a `brew cask` problem please file this issue at https://github.com/Homebrew/homebrew-cask/issues/new/choose. If it's a tap (e.g. Homebrew/homebrew-php) problem please file this issue at the tap.
- [x] ran `brew update` and can still reproduce the problem?
- [x] ran `brew doctor`, fixed all issues and can still reproduce the problem?
- [ ] ran `brew gist-logs <formula>` (where `<formula>` is the name of the formula that failed) and included the output link?
- [ ] if `brew gist-logs` didn't work: ran `brew config` and `brew doctor` and included their output with your issue?

To help us debug your issue please explain:
- What you were trying to do (and why)
brew install r
- What happened (include command output)
build failed with error:
```
gcc-5 -I/home/aznb/.linuxbrew/Cellar/zlib/1.2.11/include -I/home/aznb/.linuxbrew/Cellar/libpng/1.6.37/include/libpng16 -I/home/aznb/.linuxbrew/Cellar/libtiff/4.0.10_1/include -I. -I../../../src/include -I../../../src/include  -I/home/aznb/.linuxbrew/Cellar/zlib/1.2.11/include -I/home/aznb/.linuxbrew/Cellar/libpng/1.6.37/include/libpng16 -I/home/aznb/.linuxbrew/Cellar/pcre/8.43/include -I/home/aznb/.linuxbrew/opt/freetype/include/freetype2 -I/home/aznb/.linuxbrew/Cellar/expat/2.2.6/include -I/home/aznb/.linuxbrew/Cellar/util-linux/2.33.2/include/uuid -I/home/aznb/.linuxbrew/Cellar/fontconfig/2.13.1/include -I/home/aznb/.linuxbrew/Cellar/libffi/3.2.1/lib/libffi-3.2.1/include -I/home/aznb/.linuxbrew/Cellar/glib/2.60.1/include -I/home/aznb/.linuxbrew/Cellar/glib/2.60.1/include/glib-2.0 -I/home/aznb/.linuxbrew/Cellar/glib/2.60.1/lib/glib-2.0/include -I/home/aznb/.linuxbrew/opt/gettext/include -I/home/aznb/.linuxbrew/Cellar/pixman/0.38.4/include/pixman-1 -I/home/aznb/.linuxbrew/Cellar/libxau/1.0.9/include -I/home/aznb/.linuxbrew/Cellar/libxdmcp/1.1.2/include -I/home/aznb/.linuxbrew/Cellar/libxcb/1.13.1/include -I/home/aznb/.linuxbrew/Cellar/libx11/1.6.7/include -I/home/aznb/.linuxbrew/Cellar/libxext/1.3.3/include -I/home/aznb/.linuxbrew/Cellar/libxrender/0.9.10/include -I/home/aznb/.linuxbrew/Cellar/cairo/1.16.0/include/cairo -I/home/aznb/.linuxbrew/Cellar/xorgproto/2018.4/include -I/home/aznb/.linuxbrew/Cellar/pango/1.42.4_1/include/pango-1.0 -I/home/aznb/.linuxbrew/Cellar/harfbuzz/2.4.0_1/include/harfbuzz -I/home/aznb/.linuxbrew/Cellar/graphite2/1.3.13/include -I/home/aznb/.linuxbrew/Cellar/pango/1.42.4_1/include/pango-1.0 -I/home/aznb/.linuxbrew/Cellar/fribidi/1.0.5/include/fribidi -I../../../src/library/grDevices/src/cairo -I/home/aznb/.linuxbrew/opt/gettext/include -I/home/aznb/.linuxbrew/opt/readline/include -DHAVE_CONFIG_H   -fopenmp -fpic  -g -O2  -c devX11.c -o devX11.o
In file included from devX11.c:72:0:
devX11.h:59:27: fatal error: pango/pango.h: No such file or directory
compilation terminated.
make[4]: *** [devX11.o] Error 1
make[4]: *** Waiting for unfinished jobs....
```
- What you expected to happen
r compiles successfully 
- Step-by-step reproduction instructions (by running `brew install` commands)
brew install r

Can confirm superenv removes pango from make:
```
superenv removed:  -I/home/aznb/.linuxbrew/Cellar/libtiff/4.0.10_1/include -I/home/aznb/.linuxbrew/Cellar/pango/1.42.4_1/include/pango-1.0 -I/home/aznb/.linuxbrew/Cellar/harfbuzz/2.4.0_1/include/harfbuzz -I/home/aznb/.linuxbrew/Cellar/graphite2/1.3.13/include -I/home/aznb/.linuxbrew/Cellar/pango/1.42.4_1/include/pango-1.0 -I/home/aznb/.linuxbrew/Cellar/fribidi/1.0.5/include/fribidi -g
```
which should not have happened because R's configure script explicitly require pango:
```
configure:40656: checking whether pkg-config knows about cairo and pango
```

adding `depends_on "pango" => :build` does work

**Comment by jonchang** [comment]
> adding `depends_on "pango" => :build` does work

Can you open a pull request for this please? thanks!

**Comment by Char-Aznable** [comment]
see https://github.com/Homebrew/linuxbrew-core/pull/12987

**Comment by iMichka** [comment]
Fixed by #12987
